.github/scripts/build_assets/SeleniumRunner.py

```diff
@@ -191,45 +197,46 @@ def test_for_possible_alert(self, wait_period: float, btn_text: str):
             )
             dismiss_btn.click()
         except SeleniumTimeoutException:
-            pass
+            pass  # nothing found => everything is good
 
-    def remove_color_from_icon(self):
+    def click_on_just_added_icon(self, screenshot_folder: str, index: int):
         """
-        Remove the color from the most recent uploaded icon.
-        :return: None.
+        Click on the most recently added icon so we can remove the colors
+        and take a snapshot if needed.
         """
         try:
             recently_uploaded_icon = WebDriverWait(self.driver, self.LONG_WAIT_IN_SEC).until(
                 ec.element_to_be_clickable((By.XPATH, "//div[@id='set0']//mi-box[1]//div"))
             )
             recently_uploaded_icon.click()
-        except Exception as e:
-            self.close()
-            raise e
 
-        try:
-            color_tab = WebDriverWait(self.driver, self.SHORT_WAIT_IN_SEC).until(
-                ec.element_to_be_clickable((By.CSS_SELECTOR, "div.overlayWindow i.icon-droplet"))
-            )
-            color_tab.click()
+            self.remove_color_from_icon()
 
-            remove_color_btn = self.driver \
-                .find_element_by_css_selector("div.overlayWindow i.icon-droplet-cross")
-            remove_color_btn.click()
-        except SeleniumTimeoutException:
-            pass
-        except Exception as e:
-            self.close()
-            raise e
+            if screenshot_folder:
+                screenshot_path = str(Path(screenshot_folder, f"screenshot_{index}.png").resolve())
+                self.driver.save_screenshot(screenshot_path)
+                print("Took screenshot and saved it at " + screenshot_path)
 
-        try:
             close_btn = self.driver \
                 .find_element_by_css_selector("div.overlayWindow i.icon-close")
             close_btn.click()
         except Exception as e:
             self.close()
             raise e
 
+    def remove_color_from_icon(self):
```

Comment:
Can you explain why this method is required? Which color is beeing removed? :)

Reply:
Hello there,

So this method is to remove any colors found when we upload the svgs up to Icomoon. Sometime, the contributor would want to convert an original svg to an icon then create an alias. Instead of forcing them to remove the colors beforehand, we can use Icomoon's built-in remove color functionality.

For example, React is an icon that use its original version for the icon:
```
    {
        "name": "react",
        "tags": [
            "framework"
        ],
        "versions": {
            "svg": [
                "original",
                "original-wordmark"
            ],
            "font": [
                "original",
                "original-wordmark"
            ]
        },
```

Here's React original:
![React Original](https://raw.githubusercontent.com/devicons/devicon/master/icons/react/react-original.svg)

There are some plain icons that also have colors in their svgs as well, such as:

![Android](https://raw.githubusercontent.com/devicons/devicon/master/icons/android/android-plain.svg)

So this would allow us to have a clean icon (so it fits within one font symbol) while allowing the plain svgs to retain their colors.

Reply:
Cool! 👍🏻

Reply:
Maybe you can add this as a comment in the code?

Reply:
Yup, for sure!